ui/jobs/jobs.js

```diff
@@ -766,9 +782,10 @@ class Bars {
     if (this.buffTracker)
       this.buffTracker.clear();
     // Reset job-specific ui
-    const reset = getReset(this.job);
-    if (reset)
-      reset.bind(null, this)();
+    // const reset = getReset(this.job);
+    // if (reset)
+    //   reset.bind(null, this)();
+    this.jobComponent?.reset();
```

Comment:
Is there a reason to keep this commented out code around (same with above lines 361-363)?

Additionally, just wanted to double check that the CEF version in use by OverlayPlugin supports the `?` operator in this context?

Reply:
This is still work in progress, when I finish all the conversion, I'll delete these commented lines. Thanks for pointing out.